src/components/Alphabet/Letter/Letter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './Letter.css';
+
+const checkLetter = (letter, id, clickedLetter) => {
+    const url = `http://localhost:3000/phrases/check?id=${id}&letter=${letter}`
```

Comment:
If you are sending POST, add the parameters into params, instead of query params.
(query params are mostly for GET)

Reply:
Also possibly unnecessary localhost (see below comment)

Reply:
I think I get it ;)